README.md

```diff
@@ -490,7 +490,7 @@ If you would like to contribute to new features, check our [call for contributio
 ## Examples, tutorials and demos
 
 A series of [examples](examples/) are provided with an illustrative purpose:
-- [DQN and Rainbow](examples/dqn/dqn.py)
```

Comment:
Perhaps we want to link to the parent directory?
We mention rainbow there: is there a plan to incorporate it too? Maybe as a separate script?

Reply:
yes let's just link the pared directory and remove rainbow for now